cmd/nerdctl/create_linux_test.go

```diff
@@ -33,3 +36,69 @@ func TestCreate(t *testing.T) {
 	base.Cmd("start", tID).AssertOK()
 	base.Cmd("logs", tID).AssertOutContains("foo")
 }
+
+func TestCreateWithMACAddress(t *testing.T) {
+	base := testutil.NewBase(t)
+	tID := testutil.Identifier(t)
+	networkBridge := "testNetworkBridge" + tID
+	networkMACvlan := "testNetworkMACvlan" + tID
+	networkIPvlan := "testNetworkIPvlan" + tID
+	base.Cmd("network", "create", networkBridge, "--driver", "bridge").AssertOK()
+	base.Cmd("network", "create", networkMACvlan, "--driver", "macvlan").AssertOK()
+	base.Cmd("network", "create", networkIPvlan, "--driver", "ipvlan").AssertOK()
```

Comment:
You need to make sure to remove the network you created

Reply:
Yes, I do it in `t.Cleanup()` just lines below.